flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/enumerator/subscriber/KafkaSubscriberUtils.java

```diff
@@ -38,6 +40,19 @@ static Map<String, TopicDescription> getAllTopicMetadata(AdminClient adminClient
         }
     }
 
+    static Map<String, TopicDescription> getTopicMetadata(AdminClient adminClient, Pattern topicPattern) {
+        try {
+            Set<String> allTopicNames = adminClient.listTopics().names().get();
+            Set<String> patternTopicNames = allTopicNames.stream()
```

Comment:
nit: matchedTopic* would sound nicer than patternTopic* (not just here, in the whole pr)

Reply:
> nit: matchedTopic* would sound nicer than patternTopic* (not just here, in the whole pr)

Thank you for your careful review, I changed to them.

Reply:
> nit: matchedTopic* would sound nicer than patternTopic* (not just here, in the whole pr)

done.